content_scripts/desktopRenderer.js

```diff
@@ -146,12 +146,16 @@ import {ShaderMode} from './Shaders.js';
                     navmesh.maxX - navmesh.minX,
                     navmesh.maxZ - navmesh.minZ
                 );
+                let ceilingY = {
```

Comment:
This feels like a strange name choice for this variable, since it seems to have the ceiling and the floor?